src/app/employer/(home)/_components/Tablel.tsx

```diff
@@ -47,21 +53,22 @@ const Table: React.FC<Props> = ({ className }) => {
     useEffect(() => {
         const fetchJobs = async () => {
             try {
-                const response = await axios.get('https://carrerhub-backend.vercel.app/api/v1/jobs');
+                const response = await axios.get('https://carrerhub-backend.vercel.app/api/v1/employeer/job', {
+                    withCredentials: true // Include credentials with the request
```

Comment:
donot harcode the url, extract it to a promise and use it with tanstack/react-query